app/views/game_search/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<header class='masthead'>
+  <div class="container d-flex h-100 align-items-center">
+    <div class="mx-auto text-center">
+      <h1 class="mx-auto my-0 text-uppercase header-index">Search Results</h1></br>
+      <div class="search-results">
+      <section class="top-margin">
+        <table class = "game-table">
+            <tr>
+              <th>Save Games</th>
+              <th></th>
+              <th>Title</th>
+              <th>Rating/100</th>
+              <th>Popularity</th>
+              <th>Themes</th>
+              <th>Video</th>
+            </tr>
+          <%@games.each do |game| %>
+              <tr id="game-<%= game.id %>">
+                <td><%= button_to user_games_path(game.id), method: :post, class: "transparent save-game" do %>
+                  <i class="fa fa-gamepad fa-4x", aria-hidden="true"></i>
+                <% end %>
+                </td>
+                <td><img class="game-img thumbnail" src="<%=game.cover %>" alt=""></td>  
+                <td><%=game.title %></td>
+                <td><%=game.agg_rating %></td>
+                <td><%=game.popularity %></td>
+                <td><%=game.themes %></td>
+                <td><%=game.video %></td>
+              </tr>
+          <% end %>
+    </div>
+
+        </table>
+      </section>
+    </div>
+</header>
```

Comment:
This is currently pulling from all seeded Games in order to be able to show something on the page for now